src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -56,15 +65,41 @@
         event.target.closest('form').submit();
     };
     const toggleDisabledStateOnApplyBtn = () => {
+        let isLastModifiedRangeSelected = true;
+        let isCreatedRangeSelected = true;
         const contentTypeOption = contentTypeSelect.querySelector('option');
         const isContentTypeSelected = contentTypeOption.innerHTML !== contentTypeOption.dataset.default;
         const isSectionSelected = sectionSelect ? !!sectionSelect.value : false;
         const isModifiedSelected = !!lastModifiedSelect.value;
         const isCreatedSelected = !!lastCreatedSelect.value;
         const isCreatorSelected = !!searchCreatorInput.value;
         const isSubtreeSelected = !!subtreeInput.value.trim().length;
+
+        if (lastModifiedSelect.value === 'custom_range') {
+            const flatPickrInstance = lastModifiedDateRange._flatpickr;
+
+            if (flatPickrInstance.selectedDates.length !== 2) {
+                isLastModifiedRangeSelected = false;
+            }
+        }
+
+        if (lastCreatedSelect.value === 'custom_range') {
+            const flatPickrInstance = lastCreatedDateRange._flatpickr;
+
+            if (flatPickrInstance.selectedDates.length !== 2) {
+                isCreatedRangeSelected = false;
+            }
+        }
+
         const isEnabled =
-            isContentTypeSelected || isSectionSelected || isModifiedSelected || isCreatedSelected || isCreatorSelected || isSubtreeSelected;
+            (isContentTypeSelected ||
+                isSectionSelected ||
+                isModifiedSelected ||
+                isCreatedSelected ||
+                isCreatorSelected ||
+                isSubtreeSelected) &&
+            isLastModifiedRangeSelected &&
```

Comment:
I think it shouldn't be `&&` it should be enabled if `isLastModifiedRangeSelected` or `isCreatedRangeSelected` is filled.